keda/templates/25-metrics-poddisruptionbudget.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.podDisruptionBudget }}
```

Comment:
This is a breaking change, can you make it non-breaking please?

Reply:
Any hint how i could do that without breaking?